haas/rest.py

```diff
@@ -53,23 +53,22 @@ def rest_call(method, path, schema=None):
     * path - the url-path to map the function to. The format is the same as for
             flask's router (e.g. '/foo/<bar>/baz')
     * method - the HTTP method for the api call (e.g. POST, GET...)
-    * schema (optional) - an instance of ``schema.Schema`` with which to
-            validate the body of the request. It should assume its argument is
-            a JSON object, represented as a python ``dict``.
+    * schema - an instance of ``schema.Schema`` with which to
+            validate the arguments to the function. The schema should expect
+            a dictionary, where the keys are the names of the arguments. If
```

Comment:
Could you add after `validate the arguments to the function`: , whether specified in the URL or in the JSON body.